examples/step-74/doc/results.dox

```diff
@@ -0,0 +1,195 @@
+
+DEAL::Cycle 3
+DEAL::Number of active cells:       21
+DEAL::Number of degrees of freedom: 336
+   Solving system...
+DEAL::Writing solution to <sol_Q3-03.vtu>
+   Error in the L2 norm       :     0.000939329
+   Error in the H1 seminorm   :     0.0470787
+   Error in the enery norm    :     0.0668118
+
+DEAL::Cycle 4
+DEAL::Number of active cells:       27
+DEAL::Number of degrees of freedom: 432
+   Solving system...
+DEAL::Writing solution to <sol_Q3-04.vtu>
+   Error in the L2 norm       :     0.000339722
+   Error in the H1 seminorm   :     0.0273542
+   Error in the enery norm    :     0.0394557
+
+DEAL::Cycle 5
+DEAL::Number of active cells:       33
+DEAL::Number of degrees of freedom: 528
+   Solving system...
+DEAL::Writing solution to <sol_Q3-05.vtu>
+   Error in the L2 norm       :     0.000217819
+   Error in the H1 seminorm   :     0.0226131
+   Error in the enery norm    :     0.0324468
+
+DEAL::Cycle 6
+DEAL::Number of active cells:       42
+DEAL::Number of degrees of freedom: 672
+   Solving system...
+DEAL::Writing solution to <sol_Q3-06.vtu>
+   Error in the L2 norm       :     9.95869e-05
+   Error in the H1 seminorm   :     0.0143484
+   Error in the enery norm    :     0.0205855
+
+DEAL::Cycle 7
+DEAL::Number of active cells:       54
+DEAL::Number of degrees of freedom: 864
+   Solving system...
+DEAL::Writing solution to <sol_Q3-07.vtu>
+   Error in the L2 norm       :     6.45972e-05
+   Error in the H1 seminorm   :     0.00914956
+   Error in the enery norm    :     0.0131191
+
+DEAL::Cycle 8
+DEAL::Number of active cells:       69
+DEAL::Number of degrees of freedom: 1104
+   Solving system...
+DEAL::Writing solution to <sol_Q3-08.vtu>
+   Error in the L2 norm       :     4.16999e-05
+   Error in the H1 seminorm   :     0.00584172
+   Error in the enery norm    :     0.0083634
+
+DEAL::Cycle 9
+DEAL::Number of active cells:       87
+DEAL::Number of degrees of freedom: 1392
+   Solving system...
+DEAL::Writing solution to <sol_Q3-09.vtu>
+   Error in the L2 norm       :     2.7367e-05
+   Error in the H1 seminorm   :     0.00372588
+   Error in the enery norm    :     0.00531894
+
+degree = 3
+| cycle | cells | dofs | L2        | H1        | Energy    | Estimator |
+| 0     | 12    | 192  | 2.783e-03 | 7.490e-02 | 1.062e-01 | 3.455e-01 |
+| 1     | 15    | 240  | 1.797e-03 | 5.685e-02 | 8.154e-02 | 3.155e-01 |
+| 2     | 18    | 288  | 1.718e-03 | 5.987e-02 | 8.509e-02 | 2.582e-01 |
+| 3     | 21    | 336  | 9.393e-04 | 4.708e-02 | 6.681e-02 | 2.174e-01 |
+| 4     | 27    | 432  | 3.397e-04 | 2.735e-02 | 3.946e-02 | 2.014e-01 |
+| 5     | 33    | 528  | 2.178e-04 | 2.261e-02 | 3.245e-02 | 1.277e-01 |
+| 6     | 42    | 672  | 9.959e-05 | 1.435e-02 | 2.059e-02 | 8.352e-02 |
+| 7     | 54    | 864  | 6.460e-05 | 9.150e-03 | 1.312e-02 | 5.609e-02 |
+| 8     | 69    | 1104 | 4.170e-05 | 5.842e-03 | 8.363e-03 | 3.807e-02 |
+| 9     | 87    | 1392 | 2.737e-05 | 3.726e-03 | 5.319e-03 | 2.590e-02 |
```

Comment:
That's just too small a number of cells to really say very much. Start with 100 and show results to 10,000.